app/views/steps/applicant/has_solicitor/edit.html.erb

```diff
@@ -1,18 +1,21 @@
 <% title t('.page_title') %>
 <% heading = t(".heading", count: current_c100_application.applicants.count) %>
+<% step_header %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
 
-    <h1 class="heading-xlarge gv-u-heading-xxlarge">
-      <%= heading %>
-    </h1>
+    <h1 class="govuk-heading-xl"><%= heading %></h1>
 
-    <p class="govuk-govspeak gv-s-prose"><%=t '.lead_text' %></p>
+    <p class="govuk-body"><%=t '.lead_text' %></p>
+
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_radio_buttons_fieldset :has_solicitor, legend: { tag: 'span', size: 's' } do %>
```

Comment:
Same as above, we can use the shorter form `govuk_collection_radio_buttons` here.

And can we incorporate the heading and `lead_text` into the fieldset, like we do for example here:
steps/abuse_concerns/question/edit.html.erb